src/app/[locale]/requests/clearingRequest/detail/[id]/components/ClearingComments.tsx

```diff
@@ -10,24 +10,108 @@
+            return null
+        }
+        const date = new Date(timestamp)
+        const year = date.getFullYear()
+        const month = String(date.getMonth() + 1).padStart(2, '0')
+        const day = String(date.getDate()).padStart(2, '0')
+        const hours = String(date.getHours()).padStart(2, '0')
+        const minutes = String(date.getMinutes()).padStart(2, '0')
+        const seconds = String(date.getSeconds()).padStart(2, '0')
+    
+        return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`
+    }
+
+    const fetchData = useCallback(
+        async (url: string) => {
+            const response = await ApiUtils.GET(url, session.user.access_token)
+            if (response.status == HttpStatus.OK) {
+                const data = await response.json() as EmbeddedClearingRequestComments
+                return data
+            } else if (response.status == HttpStatus.UNAUTHORIZED) {
+                return signOut()
+            } else {
+                notFound()
+            }
+        },[session]
+    )
+
+    useEffect(() => {
+        setLoading(true)
+        void fetchData(`clearingrequest/${clearingRequestId}/comments`).then(
+                    (clearingRequestCommentList: EmbeddedClearingRequestComments) => {
+            if (
+                !CommonUtils.isNullOrUndefined(clearingRequestCommentList['_embedded']) &&
+                !CommonUtils.isNullOrUndefined(clearingRequestCommentList['_embedded']['sw360:comments'])
+            ){
+                setComments(clearingRequestCommentList['_embedded']['sw360:comments'])
+                setLoading(false)
+            }
+        })
+        .catch((err) => console.error(err))
+    }, [fetchData, session])
+
+    const updateInputField = (event: React.ChangeEvent<HTMLSelectElement |
+                                     HTMLInputElement |
+                                     HTMLTextAreaElement>) => {
+        setInputComment(event.target.value)
+        setCommentPayload({
+            ...commentPayload,
+            [event.target.name]: event.target.value,
+        })
+    }
 
-    const handleAddComment = () => {
-        console.log('Test')
+    const handleAddComment = async () => {
+        const response = await ApiUtils.POST(`clearingrequest/${clearingRequestId}/comments`,
+                                              commentPayload,
+                                              session.user.access_token)
+        if (response.status == HttpStatus.OK) {
+            const response_data = await response.json() as EmbeddedClearingRequestComments
+            setInputComment('')
+            setComments(response_data._embedded['sw360:comments'])
+            MessageService.success(t('Your comments updated successfully'))
+        } else if (response.status == HttpStatus.UNAUTHORIZED) {
+            return signOut()
+        } else {
+            MessageService.error(t('There are some problem to update your comments'))
+            notFound()
```

Comment:
I think we don't need notFound here

Reply:
Thanks @hoangnt2 for pointing out. I forgot to remove it.
Now it is fixed ! 👍🏻